editor/editor_inspector.cpp

```diff
@@ -963,6 +963,7 @@ EditorProperty::EditorProperty() {
 	selected_focusable = -1;
 	label_reference = nullptr;
 	bottom_editor = nullptr;
+	delete_hover = false;
```

Comment:
All those should be moved to the header (but that's maybe out of scope of this PR, see #38697).